### PR TITLE
frieren: volume-points curriculum 16k->65k on SOTA stack

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

Volume tokens currently train at the full 65k points/sample from epoch 1. With ~12-epoch training budget and Transolver's slice-attention scaling with token count, the optimizer spends early epochs propagating gradients through a maximally-noisy 65k-point context. **Curriculum on volume point count** — ramp from `vol_pts ∈ {16384, 32768, 49152, 65536}` over the first 4 epochs — gives early gradients a smaller, denser sample of the volume domain to converge on, then expands to the full point budget once basic structure is learned.

This is analogous to image-resolution curriculum in vision (low-res → high-res), and to Petrov-Galerkin coarse-to-fine in numerical methods. Surface points stay at 65k throughout (they're already the SOTA setup).

## Instructions

1. **Add CLI flag `--volume-points-curriculum` (default empty/None)** in `target/train.py`. Parse format `"epoch:n_points,epoch:n_points,..."` e.g. `"0:16384,1:32768,2:49152,3:65536"`.

2. **In the training loop / dataloader setup,** at the start of each epoch, look up the current volume-points target from the curriculum schedule and update the dataloader's `train_volume_points` for that epoch. Validation/test always use 65536 (full).

3. **Be careful with sampler/dataloader recreation across epochs** — DDP samplers must be re-seeded per epoch. The simplest path: tear down and rebuild the train DataLoader at epoch start when the curriculum step changes; reuse otherwise.

4. **Keep surface points at 65536 throughout.** Only volume points change.

**Reproduce command (DDP8, on SOTA stack):**

```bash
torchrun --standalone --nproc_per_node=8 train.py \
  --agent frieren --optimizer lion --lr 1e-4 --weight-decay 5e-4 \
  --no-compile-model --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --model-layers 4 --model-hidden-dim 512 --model-heads 4 --model-slices 128 \
  --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 \
  --pos-encoding-mode string_separable --use-qk-norm --rff-num-features 16 \
  --volume-points-curriculum "0:16384,1:32768,2:49152,3:65536" \
  --wandb-group frieren-vol-curriculum \
  --wandb-name frieren/volume-points-curriculum-4step
```

## Baseline

- SOTA to beat: PR #387 alphonse, **val_abupt=7.3816%** (EP11), test=8.5936%.
- Volume_pressure is the #1 laggard at test=12.19% vs AB-UPT 6.08%.
- Smaller early volume context means faster epoch wallclock for EP1-3 (extra epochs of training within the 270-min budget if compute drops). Verify epoch wallclock decreases for the curriculum-low epochs.
- Expected lift: -0.3 to -1.5pp. Curriculum on token count is well-validated in vision (progressive resizing) and in Transformer pretraining (Vaswani's variable-length curriculum).

If this wins, it composes orthogonally with everything else (architecture changes, EMA, optimizer) and could become a SOTA-stack default.

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #484 because Closing this draft to resolve the duplicate assignment conflict for frieren. Frieren already has an active retry task on PR #454 (tau_yz weight 1.5x). The volume-points curriculum hypothesis in this PR is promising and will be reassigned to a different student when a slot opens up. The hypothesis text is preserved here for reference..
